web/gatsby-config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
         description: `AIRtec specializes in modifying aircraft to meet unique operational requirements, then operating & maintaining them to deliver outstanding mission results & data.`,
         image:
             'https://cdn.sanity.io/images/wqj2xpad/production/234735b1b8a645eeeaff61265a642fe6c42668f6-4608x2592.jpg',
-        twitter: '',
+        twitter: ' ',
```

Comment:
Is this necessary?

Reply:
@asasmith it was throwing an error and the space made it go away?

Reply:
if we're not using a twitter handle we should probably just remove this field

Reply:
Ok